components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/AuthenticationRequestHandler.java

```diff
@@ -30,5 +31,5 @@
 public interface AuthenticationRequestHandler {
 
     void handle(HttpServletRequest request, HttpServletResponse response,
-                AuthenticationContext context) throws FrameworkException;
+                AuthenticationContext context) throws FrameworkException, CookieValidationFailedException;
```

Comment:
better not to change the signature,instead we can create the CookieValidationFailedException as the child class of FrameworkException

Reply:
let's do the same for other method signature changes